firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -377,6 +383,22 @@ public static void setLoggingEnabled(boolean loggingEnabled) {
     }
   }
 
+  /**
+   * Clears the persistent storage.
+   *
+   * <p>Must be called while the client is not started (after the app is shutdown or when the app is
+   * first initialized). On startup, this method must be called before other methods (other than
+   * settings()). If the client is still running, an exception with a code of `failed-precondition`
+   * will be thrown.
+   */
+  @PublicApi
```

Comment:
This would release the feature. For now, remove `@PublicApi` and `public` to make this package private.